tests/sweep_framework/sweeps/eltwise/unary/isinf/isinf_sharded.py

```diff
@@ -29,7 +35,7 @@
 # Developers can create their own generator functions and pass them to the parameters as inputs.
 parameters = {
     "nightly": {
-        "input_spec": gen_sharded_spec_unary(16, layouts=["TILE_LAYOUT"]),
```

Comment:
Why is  layouts=["TILE_LAYOUT"] removed?

Reply:
I modified existing `gen_sharded_spec_unary` as well as added invalidation for page size error when using ROW_MAJOR_LAYOUT so that now we can generate some parameter vectors with ROW_MAJOR_LAYOUT.